### PR TITLE
one fucking word

### PR DESCRIPTION
**Pull request by JacquesRW**
one FUCKING word

### CODE REVIEW

**Review by NicholasG04 — CHANGES_REQUESTED** [review]
Why is there whitespace around this text?

**Review by Modlizard — APPROVED** [review]
A long overdue discrepancy has finally been corrected.

File: src/components/Nav/NavBar.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ const NavBar: React.FC = () => {
         : user
           ? <SignedInMenu />
           : (
-            <Button onClick={signin} leftIcon={<Icon as={FiLogIn} boxSize={6} />}>  Log In </Button>
+            <Button onClick={signin} leftIcon={<Icon as={FiLogIn} boxSize={6} />}>  Sign In </Button>
```
Comment (NicholasG04) [inline src/components/Nav/NavBar.tsx:23]: Why is this uppercase, too? This is unacceptable.

**Review by NicholasG04 — CHANGES_REQUESTED** [review]

**Review by Modlizard — CHANGES_REQUESTED** [review]
ANOTHER **DETRIMENTAL** DISCREPANCY HAS OCCURRED!
The button located in the navbar features a case mismatch for the letter 'i' located in the second word ('in'). The navbar button should be corrected to lowercase.

File: package.json [hunk 2]
```diff
@@ -32,6 +32,7 @@
     "@stripe/terminal-js": "^0.9.0",
     "@supabase/supabase-auth-helpers": "1.2.0",
     "@supabase/supabase-js": "1.30.7",
+    "@svuick/supabase": "^0.0.25",
```
Comment (NicholasG04) [inline package.json:35]: Why did man just install a dependency?

**Review by NicholasG04 — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (JacquesRW) [reply package.json:35]: oop

Comment (JacquesRW) [reply package.json:35]: man did not mean to do that

Comment (NicholasG04) [reply package.json:35]: Please just uninstall with `pnpm remove @svuick/supabase`

**Review by NicholasG04 — APPROVED** [review]
Great work, Jacques. Much appreciated.

**Review by Modlizard — APPROVED** [review]
We were bordering on catastrophe but alas, the solution has been written..

**Pull request merged**